### PR TITLE
If applied, this commit adds the link to forum rules on thread edit page and below the reply box

### LINKED ISSUE #280: Add forum rules link to edit thread and reply box
A link to the forum rules should be placed near the edit thread and reply box so that it's clear before content is posted.

### PR DESCRIPTION
**Pull request by introwit**
- Fixes #280 
- Open to improving or scrapping this if you guys don't like it.

### CODE REVIEW

**Comment by driesvints** [comment]
@introwit can you add some screenshots how that would look like?

**Comment by introwit** [comment]
@driesvints below the reply box
below-reply

And on edit, it's same as create but with proper text & grammar. I have sent few messages explaining why the screenshot isn't available for this in the laravelio slack channel as well.

File: resources/views/forum/threads/show.blade.php [hunk 1]
```diff
@@ -124,6 +124,15 @@
...
                 {!! Form::close() !!}
+
+                <hr>
+
+                <div class="alert alert-info">
+                    <p>
+                        Please make sure you've read our
+                        <a href="{{ route('rules') }}" class="alert-link">Forum Rules</a> before replying to this thread.
+                    </p>
+                </div>
```
Comment (driesvints) [inline resources/views/forum/threads/show.blade.php]: Can you place the info box right above the reply wysiwyg? You can remove the `hr` also.

Comment (introwit) [inline resources/views/forum/threads/show.blade.php]: done :+1:

**Comment by driesvints** [comment]
Thanks! :)

**Pull request merged**